lxd/firewall/drivers/drivers_xtables.go

```diff
@@ -1370,6 +1370,56 @@ func (d Xtables) InstanceClearRPFilter(projectName string, instanceName string,
 	return nil
 }
 
+// InstanceSetupNetPrio activates setting of skb->priority for the specified instance device on the host interface.
+func (d Xtables) InstanceSetupNetPrio(projectName string, instanceName string, deviceName string, netPrio int) error {
```

Comment:
shall we make netPrio uint32 to make it clear thats the supported range (as per the validation rules later in this PR)?

Reply:
Done.